content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md

```diff
@@ -0,0 +1,149 @@
+---
+author: hiddeco
+date: 2023-12-12 16:00:00+00:00
```

Comment:
Let's set this at 14

Reply:
I think 15 should be fine, given it's UTC.